dmaws/commands/exportdata.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+import click
+
+from ..cli import cli_command
+from ..stacks import StackPlan
+from ..rds import RDS, RDSPostgresClient
+
+
+@cli_command('exportdata', max_apps=0)
+@click.option('--export-path', default='.',
+              help="Path to write export files.")
+def exportdata_cmd(ctx, export_path):
+    plan = StackPlan.from_ctx(ctx, apps=['database_dev_access'])
+
+    if not plan.create(create_dependencies=False):
```

Comment:
Do we need dev access to production to take a snapshot or is this to avoid manually creating security groups for snapshot DB?

I'm a bit worried that the difference between cleaning up a snapshot and wiping out production data is `tmp_` in the `RDSPostgresClient` argument.